src/prefect/cli/orion.py

```diff
@@ -150,6 +153,54 @@ async def start(
     app.console.print("Orion stopped!")
 
 
+@orion_app.command()
+async def status(
+    wait: bool = typer.Option(
+        False,
+        "--wait",
+        help="When the wait command is specified the health check loop runs until successful or an optional timeout value is exceeded, otherwise the check is run only once.",
+    ),
+    timeout: int = typer.Option(
+        None,
+        "--timeout",
+        help="Optional Timeout value in seconds, must be used with the wait argument",
+    ),
+    api: str = SettingsOption(PREFECT_API_URL),
+):
+    """Verify Connection status with Orion API"""
+
+    if not api:
+        exit_with_success(
+            "PREFECT_API_URL not set for the currently active profile. To check the status of an Orion instance or Cloud API connection the PREFECT_API_URL must be specified for the selected profile."
```

Comment:
```suggestion
            "PREFECT_API_URL not set for the currently active profile. To check the status of an Orion instance or Cloud API connection, the PREFECT_API_URL must be specified for the selected profile."
```

Reply:
Thanks Terry I updated the messages for each of these.